charts/em-stitching/values.yaml

```diff
@@ -21,10 +21,15 @@ stitching:
         - AppInsightsInstrumentationKey
         - docmosis-access-key
         - microservicekey-em-stitching-api
+        - stitching-POSTGRES-DATABASE
+        - stitching-POSTGRES-HOST
+        - stitching-POSTGRES-PASS
+        - stitching-POSTGRES-PORT
+        - stitching-POSTGRES-USER
   environment:
-    SPRING_DATASOURCE_URL: jdbc:postgresql://{{ .Release.Name }}-{{ .Values.postgresql.nameOverride}}:5432/postgres?ssl=disable
```

Comment:
these values need to be moved to values.preview.template.yaml:
https://github.com/hmcts/rpa-em-stitching-api/blob/master/charts/em-stitching/values.preview.template.yaml